strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/providers/layout/Maven2LayoutProvider.java

```diff
@@ -46,15 +47,14 @@
  * @author carlspring
  */
 @Component("maven2LayoutProvider")
-public class Maven2LayoutProvider extends AbstractLayoutProvider<MavenArtifactCoordinates,
-                                                                 MavenRepositoryFeatures,
-                                                                 MavenRepositoryManagementStrategy>
+public class Maven2LayoutProvider
```

Comment:
Ok, but I re-formatted the code using provided [strongbox-idea.xml](https://raw.githubusercontent.com/wiki/strongbox/strongbox/resources/codestyles/strongbox-idea.xml)  ;)